package.json

```diff
@@ -7,7 +7,10 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && ./node_modules/.bin/babel -sd lib/ src/",
```

Comment:
I found that if I didn't do that, it would use the system babel, which is weird and doesn't make much sense to me.

The -s flag is killing me, maybe a second set of eyes would help.  If I add sourceMaps: true to the babelrc file, it doesn't generate the mapping or add the magic comment to the output.  If I set it to inline, it does generate a source map in the generated file, but the stacks still don't work.

The only way I can get a .map file is to use -s, and the stacks seem to be messed up right now.  If you have ideas, I'm all ears :)